### PR TITLE
Redirect log in the tomcat log file

### PR DESCRIPTION
**Pull request by javamind**
If the webapp is not deployed in Tomcat, the logger is the console

### CODE REVIEW

File: backend/webapp/pom.xml [hunk 1]
```diff
@@ -162,6 +162,11 @@
             <version>1.1.2</version>
         </dependency>
         <dependency>
+            <groupId>org.codehaus.janino</groupId>
```
Comment (cpuy) [inline backend/webapp/pom.xml]: Maybe a small comment here to explain why we have this dependency would be nice

File: backend/webapp/src/main/resources/logback.xml [hunk 2]
```diff
@@ -1,15 +1,35 @@
...
+
+            <root level="INFO">
+                <appender-ref ref="CONSOLE"/>
+            </root>
+        </else>
+    </if>
 
-    <root level="INFO">
-        <appender-ref ref="CONSOLE" />
-    </root>
```
Comment (cpuy) [inline backend/webapp/src/main/resources/logback.xml]: Would it be better to define appender with same name and let the following block outside of the if/else ?
```xml
<root level="INFO">
       <appender-ref ref="MYAPPENDER"/>
</root>
```

**Comment by cpuy** [comment]
:+1: 
IMHO this is a quite elegant solution. Good work.
Please see comment above

**Comment by cpuy** [comment]
In standalone jar mode, we don't have logs anymore in the console (.extract/logs/ui-designer.log)
Would it be acceptable to have LOGFILE + CONSOLE ?

File: backend/webapp/src/main/resources/logback.xml (hunk 2 above)
Comment (javamind) [inline backend/webapp/src/main/resources/logback.xml]: It's possible

**Comment by javamind** [comment]
For the standalone mode, why not but it's difficult to know that we are in this mode. We could use an env variable

**Comment by cpuy** [comment]
:+1:  Would you mind squashing your commits ?

**Pull request merged**